plugins/extractit/plugin.spec.yaml

```diff
@@ -121,6 +121,13 @@ actions:
         type: bytes
         required: false
         example: aHR0cHM6Ly9leGFtcGxlLmNvbSBpcyBhbiBleGFtcGxlIFVSTA==
+      keep_original_urls:
+        title: Keep Original URLs
+        description: "Retain original URL encoding, if applicable. e.g: If true: http://foo.bar/?q=Test%20URL-encoded%20stuff will be extracted as http://foo.bar/?q=Test%20URL-encoded%20stuff. If false, http://foo.bar/?q=Test%20URL-encoded%20stuff will be extracted as http://foo.bar/?q=Test as %20 is decoded as a space"
```

Comment:
```suggestion
        description: "Retain original URL encoding, if applicable, e.g. if true: http://foo.bar/?q=Test%20URL will be extracted as http://foo.bar/?q=Test%20URL. If false, http://foo.bar/?q=Test%20URL will be extracted as http://foo.bar/?q=Test because %20 is decoded as a space."
```